### ISSUE TITLE
TypeScript Support

### ISSUE THREAD

**Issue by AlexBroadbent** [Treblle/treblle-fastify#2]
<img width="1319" alt="Screenshot 2023-10-05 at 14 06 37" src="https://github.com/Treblle/treblle-fastify/assets/3311606/fafe7b6e-de34-477d-8601-82f9d98301c0">

**Comment by ChukwunonsoFrank** [comment]
hey @DominusKelvin 

I could pickup on type support for the SDK.

**Comment by DominusKelvin** [comment]
Nice idea but you'd have to look at treblle-utils and how Types are exported

**Comment by ChukwunonsoFrank** [comment]
yeah. I just went through it this afternoon and would reference it for sure. I'm also doing a bit of light reading on declaration files.
